packages/legacy-scripting-runner/test/integration-test.js

```diff
@@ -756,6 +756,37 @@ describe('Integration Test', () => {
       });
     });
 
+    it('KEY_pre_poll, array curlies', () => {
+      const appDef = _.cloneDeep(appDefinition);
+      appDef.legacy.scriptingSource = appDef.legacy.scriptingSource.replace(
+        'movie_pre_poll_array_curlies',
+        'movie_pre_poll'
+      );
+      appDef.legacy.scriptingSource = appDef.legacy.scriptingSource.replace(
+        'movie_post_poll_make_array',
+        'movie_post_poll'
+      );
+      const compiledApp = schemaTools.prepareApp(appDef);
```

Comment:
not a change here, but most of these tests go through these same setup steps. I wonder if it's worth moving setup into a callable function

Reply:
Yep, that's something I wanted to do but it's never been a priority (or don't have much motivation).